torchvision/transforms/transforms.py

```diff
@@ -1225,15 +1215,13 @@ class RandomErasing(object):
             If a str of 'random', erasing each pixel with random values.
          inplace: boolean to make this transform inplace. Default set to False.
 
-    Returns:
-        Erased Image.
-    # Examples:
+    Example:
         >>> transform = transforms.Compose([
-        >>> transforms.RandomHorizontalFlip(),
-        >>> transforms.ToTensor(),
-        >>> transforms.Normalize((0.485, 0.456, 0.406), (0.229, 0.224, 0.225)),
-        >>> transforms.RandomErasing(),
-        >>> ])
+        ...     transforms.RandomHorizontalFlip(),
```

Comment:
Does this work if we mix `>>>` and `...` ?

Reply:
Yes, this is actually how doctests should be written. This also mimics the python REPL.

(Note that the indentation was of in the earlier version.)

Just to verify that it's valid here is a minimal example:
```python
class Foo:
    """Foooo

    Examples:
        >>> foo = Foo(
        ...     name="Alice"
        ... )
        >>> foo.foo()
        Alice
    """
    def __init__(self, name):
        self.name = name

    def foo(self):
        print(self.name)
```
Then
```bash
pytest --doctest-modules dummy.py
```